code/game/shuttle_engines.dm

```diff
@@ -81,18 +82,21 @@
 			M.alter_engines(mod)
 
 /obj/structure/shuttle/engine/heater
-	name = "heater"
+	name = "engine heater"
 	icon_state = "heater"
+	desc = "Directs energy into compressed particles in order to power engines."
 	engine_power = 0 // todo make these into 2x1 parts
 
 /obj/structure/shuttle/engine/platform
-	name = "platform"
+	name = "engine platform"
 	icon_state = "platform"
+	desc = "A platform for engine components, or something."

```

Comment:
no need for the `or something`

Reply:
Reason I said it that way is because it's not really clear what it actually is for, so it's kind of a meta reference. I can change it